### PR TITLE
fixed some syntax error

### PR DESCRIPTION
**Pull request by louyiping**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

**Comment by louyiping** [comment]
change

**Comment by thockin** [comment]
I appreciate the contribution, but I actually think changes are not needed.  Each case is correct, though subtle if English is not your first language.  I'll comment on them each.

File: docs/design/access.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ that make up a K8s cluster.
 
 ### Threats
 
-Both intentional attacks and accidental use of privilege are concerns.
+Both intentional attacks and accidental use of privilege are concerned.
```
Comment (thockin) [inline docs/design/access.md]: "concerns" is right, meaning "sources of concern"

File: docs/design/access.md [hunk 2]
```diff
@@ -119,7 +119,7 @@ Related discussion:
 
 This doc describes two security profiles:
   - Simple profile:  like single-user mode. Make it easy to evaluate K8s
-without lots of configuring accounts and policies.  Protects from unauthorized
+without lots of configuring accounts and policies.  Protect from unauthorized
```
Comment (thockin) [inline docs/design/access.md]: While it is an incomplete sentence, "Protects from" has an implied "it" or "this" subject.

File: docs/design/access.md [hunk 3]
```diff
@@ -130,7 +130,7 @@ simple and enterprise profiles.  System should be flexible enough for
 knowledgeable users to create intermediate profiles, but K8s developers should
 only reason about those two Profiles, not a matrix.
 
-Features in this doc are divided into "Initial Feature", and "Improvements".
+Features in this doc are divided into "Initial Features", and "Improvements".
```
Comment (thockin) [inline docs/design/access.md]: This change is OK, but the doc has moved to the community repo, so we can't take this PR anyway.

File: docs/design/access.md [hunk 4]
```diff
@@ -202,7 +202,7 @@ Improvements:
 This may provide some defense-in-depth against container escapes. (https://github.com/docker/docker/pull/4572)
 - requires docker to integrate user namespace support, and deciding what
 getpwnam() does for these uids.
-- any features that help users avoid use of privileged containers
+- any features that help users avoid using of privileged containers
```
Comment (thockin) [inline docs/design/access.md]: "use of" is correct, but "using" (without the "of") would also be OK.  No change needed

File: docs/design/access.md [hunk 5]
```diff
@@ -241,7 +241,7 @@ accounts, and no https proxy required.
 allow integration with existing to enterprise authorization systems. The
 Kubernetes namespace itself should avoid taking contributions of multiple
 authorization schemes. Instead, a trusted proxy in front of the apiserver can be
-used to authenticate users.
+used to authenticated users.
```
Comment (thockin) [inline docs/design/access.md]: This is a verb here - "to authenticate".

**Pull request closed without merging**